pkg/server/plugin/zmq/broker.go

```diff
@@ -48,23 +50,37 @@ const (
 	// It is an addition to original PPP to shorten the time needed for
 	// broker to detect a normal shutdown of worker.
 	Shutdown = "\003"
+	Request = "REQ"
+	Response = "RES"
 )
 
-// parcel is used to multiplex the chan with zmq worker
-type parcel struct {
-	respChan chan []byte
-	frame    []byte
+// Parcel is used to multiplex the chan with zmq worker
+type Parcel struct {
+	Worker   string
+	RespChan chan []byte
+	Frame    []byte
 	retry    int
 }
 
-func newParcel(frame []byte) *parcel {
-	return &parcel{
-		respChan: make(chan []byte),
-		frame:    frame,
+func newParcel(frame []byte) *Parcel {
+	return &Parcel{
+		Worker:   "",
+		RespChan: make(chan []byte),
+		Frame:    frame,
 		retry:    0,
 	}
 }
 
+var requestIDPrefix = ""
```

Comment:
I seems to have misunderstanding on how should a request-id looks like. I followed [python's code](https://github.com/SkygearIO/py-skygear/blob/5e769490051038837c7050019fe7fabb49010c2e/skygear/transmitter/zmq.py#L49) where the first part is a global constant.

Wait... in python it is per worker constant... I will make it per worker in Golang too, so it is more consistent